espanol/artefacto/artefacto.md

```diff
@@ -0,0 +1,3 @@
+Beatito asombrado cogió el artefacto, mientras el Consejero le decia "Ya no tenemos tiempo, ahora solo nos queda activarlo". A lo que Beatito 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.